spikeinterface/sorters/runsorter.py

```diff
@@ -337,6 +337,8 @@ def run_sorter_container(
             raise ValueError(f"sorter {sorter_name} not in SORTER_DOCKER_MAP. Please specify a container_image.")
 
     SorterClass = sorter_dict[sorter_name]
+    default_params = SorterClass.default_params()
+    sorter_params = {**default_params, **sorter_params}
```

Comment:
This writting is smart but hide a bit the fact that sorter_params is updating the default.